projects/workflows-creator/src/lib/builder/group/group.component.ts

```diff
@@ -239,7 +237,7 @@ export class GroupComponent<E> implements OnInit, AfterViewInit {
             this.dateTime = value;
             break;
           case InputTypes.People:
-            this.selectedItems = value;
+            this.selectedItems = value.map((item: any) => item.id);
```

Comment:
avoid using `any` type, please confirm with @yeshamavani about the type.

I believe `value` should be `[]{id: string}`

Reply:
done